readme - making download_dependencies scripts.txt

```diff
@@ -0,0 +1,12 @@
+(Re)building buildUpdate.sh requires ruby and https://github.com/chrisvire/BuildUpdate
```

Comment:
Good idea to add these instructions. Although each generated file also has a link to chrisvire/BuildUpdate, which also has these instructions, so I feel this readme is redundant.